frontend/frontend/src/app/add-employee/add-employee.component.html

```diff
@@ -1,4 +1,4 @@
-<div class="add_employee_bg">
+<div class="add_employee_bg" *ngIf="roles=='RMG Admin'">
```

Comment:
please change the variable names to camelcase without any space

Reply:
updated variable name